-1 
Tested at: 063a1656d98db6228bd7f937b6ac17dd98736c30
 I found an error when building: 
 
 <pre>                 from /afs/cern.ch/cms/sw/ReleaseCandidates/vol0/slc6_amd64_gcc493/external/geant4/10.01.p02/include/Geant4/G4GDMLReadSetup.hh:47,
                 from /afs/cern.ch/cms/sw/ReleaseCandidates/vol0/slc6_amd64_gcc493/external/geant4/10.01.p02/include/Geant4/G4GDMLReadParamvol.hh:43,
                 from /afs/cern.ch/cms/sw/ReleaseCandidates/vol0/slc6_amd64_gcc493/external/geant4/10.01.p02/include/Geant4/G4GDMLReadStructure.hh:46,
                 from /afs/cern.ch/cms/sw/ReleaseCandidates/vol0/slc6_amd64_gcc493/external/geant4/10.01.p02/include/Geant4/G4GDMLParser.hh:43,
                 from /tmp/cmsbuild/workspace/ib-any-integration/CMSSW_7_6_X_2015-09-18-1100/src/SimG4Core/Geometry/src/G4CheckOverlap.cc:13:
/afs/cern.ch/cms/sw/ReleaseCandidates/vol0/slc6_amd64_gcc493/external/geant4/10.01.p02/include/Geant4/G4GDMLRead.hh:42:47: fatal error: xercesc/parsers/XercesDOMParser.hpp: No such file or directory
 #include <xercesc/parsers/XercesDOMParser.hpp>
                                               ^
compilation terminated.
gmake: *** [tmp/slc6_amd64_gcc493/src/SimG4Core/Geometry/src/SimG4CoreGeometry/G4CheckOverlap.o] Error 1
>> Compiling  /tmp/cmsbuild/workspace/ib-any-integration/CMSSW_7_6_X_2015-09-18-1100/src/SimG4Core/Geometry/src/SensitiveDetectorCatalog.cc 
</pre> 
 you can see the results of the tests here: 
 https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-11366/8212/summary.html

--- a/SimG4Core/Geometry/BuildFile.xml
+++ b/SimG4Core/Geometry/BuildFile.xml
@@ -3,6 +3,7 @@
 <use   name="Geometry/Records"/>
 <use   name="FWCore/MessageLogger"/>
 <use   name="FWCore/Utilities"/>
+<use   name="xerces-c"/>
 <use   name="expat"/>
 <export>
   <lib   name="1"/>

